src/App.jsx

```diff
@@ -2,16 +2,16 @@ import React, { useState } from 'react';
 import './App.css';
 import { Container,  BoardGrid, Square } from './2DArray.styled';
 import { shipArray } from './renderCoordinates.helpers';
```

Comment:
opet, shipArray je const, to ne bi trebalo biti u helpers fileu.
a sam helpers slobodno stavi u helpers/ folder, trebat će ti za dalje